app/views/backend/affairs/_show.html.haml

```diff
@@ -47,7 +47,7 @@
               %td.name
                 - if deal.is_a? DebtTransfer
                   - foreign_deal = deal.debt_transfer_affair
-                  = foreign_deal.deals.reject{ |d| d.is_a?(DebtTransfer) }.map { |d| link_to(d.number, controller: "/backend/" + d.class.name.tableize, action: :show, id: d.id) }.to_sentence
+                  = raw(foreign_deal.deals.reject{ |d| d.is_a?(DebtTransfer) }.map { |d| link_to(d.number, controller: "/backend/" + d.class.name.tableize, action: :show, id: d.id) }.to_sentence)
```

Comment:
Could you use '.html_safe' instead to be more cohesive with the rest of the codebase ? (If you could remove the commented out line underneath too while you're at it it'd be great !)